maths/polynomial_evaluation.py

```diff
@@ -0,0 +1,26 @@
+def evaluate_poly(poly, x):
+    """
+        Objective: Computes the polynomial function for a given value x. 
+                Returns that value. 
+        Input Prams: 
+            poly: tuple of numbers - value of cofficients
+            x: value for x in f(x)
+        Return: value of f(x)
+    """
+    i = 0
+    res = 0
+    for c in poly:
```

Comment:
__for i, c in enumerate(poly):__  # then delete lines 14 and 10

Reply:
Actually, the whole function can be a one-liner: __return sum(c * x ** i for i, c in enumerate(poly))__